### PR TITLE
Add bit more contributing info & history of owners

### PR DESCRIPTION
**Pull request by 6543**
ref  #633

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -26,5 +26,20 @@ This can also happen when a owner propose a vote or the majority of the maintain
...
 
-For security reasons, Owners must use 2FA.
-(https://docs.github.com/en/github/authenticating-to-github/securing-your-account-with-two-factor-authentication-2fa)
+For security reasons, Owners must use 2FA.  
+([Docs: Securing your account with two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa))
+
+To honor the past owners, here's the history of the owners and the time
+they served:
+
+* 2021-xx-xx ~
```
Comment (6543) [inline CONTRIBUTING.md:35]: I'll need to lookup that in discord ;)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,5 +26,20 @@ This can also happen when a owner propose a vote or the majority of the maintain
...
+  * [6543](https://github.com/6543)
+  * [Anbraten](https://github.com/anbraten)
+
+* 20xx-xx-xx ~ 2021-xx-xx
```
Comment (6543) [inline CONTRIBUTING.md:39]: start from **on 25 Jul 2019** if we see 86f36dd744d2365d1f50cd7a91aa60c263d24d6c as beginning

File: CONTRIBUTING.md (hunk 1 above)
Comment (6543) [reply CONTRIBUTING.md:35]: 28.09.2021

Comment (6543) [inline CONTRIBUTING.md:35]:
```suggestion
* 2021-09-28 ~
```

File: CONTRIBUTING.md (hunk 2 above)
Comment (6543) [inline CONTRIBUTING.md:39]:
```suggestion
* 2019-07-25 ~ 2021-09-28
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -26,5 +26,20 @@ This can also happen when a owner propose a vote or the majority of the maintain
...
 
-For security reasons, Owners must use 2FA.
-(https://docs.github.com/en/github/authenticating-to-github/securing-your-account-with-two-factor-authentication-2fa)
+For security reasons, Owners must use 2FA.  
+([Docs: Securing your account with two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa))
+
+To honor the past owners, here's the history of the owners and the time
+they served:
+
+* 2021-09-28 ~
```
Comment (6543) [inline CONTRIBUTING.md:35]:
```suggestion
* 2021-09-28 ~ 2022-12-31 - https://github.com/woodpecker-ci/woodpecker/issues/633
```

**Review by anbraten — APPROVED** [review]

**Review by laszlocph — APPROVED** [review]

**Pull request merged**